profiler/test/Datadog.Profiler.Native.Tests/packages.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="gmock" version="1.11.0" targetFramework="native" />
-  <package id="libdatadog" version="13.1.0" targetFramework="native" />
   <package id="Microsoft.googletest.v140.windesktop.msvcstl.static.rt-static" version="1.8.1.7" targetFramework="native" />
```

Comment:
> Was this the only nuget requirement we had on the native side do you know
oh, boo, I guess not 😄 Just wondering outloud can/should we switch to vcpkg dependencies for these too? Would that make sense?

For context, NuGet with native side is a _bit_ of a pain, because you can't use the built-in `dotnet` version, you have to use the standalone NuGet etc, which adds a bit of extra complexity in places. Not a big deal, not for this PR, just wondering out loud

Reply:
yeah I remember 😓